types/rascal/index.d.ts

```diff
@@ -178,7 +178,25 @@ export interface BrokerConfig {
     recovery?: {
         [key: string]: Recovery | Recovery[];
     };
-    defaults?: BrokerConfig;
+    defaults?: {
+        vhosts?: {
```

Comment:
Instead of copying down the type of `VhostConfig`, can this just be `vhosts?: VhostConfig`?

Reply:
shouldn't it be 
` vhosts?: {
        [key: string]: VhostConfig;
    }`
??

Reply:
I don't think so - it looks like the `defaults.vhosts` property takes a single `VhostConfig` to use as a default for multiple. I don't use the library, though - I'm just basing that on the documentation [here](https://github.com/guidesmiths/rascal#defaults).

Reply:
Oh, my bad, thanks.